app/assets/javascripts/discourse/app/widgets/hamburger-menu.js

```diff
@@ -331,7 +347,27 @@ export default createWidget("hamburger-menu", {
     return results;
   },
 
-  html() {
+  refreshReviewableCount(state) {
+    const { currentUser } = this;
+
+    if (state.loading || !currentUser) return;
+
+    state.loading = true;
+
+    ajax("/review/count.json")
+      .then(({ count }) => currentUser.set("reviewable_count", count))
+      .finally(() => {
+        state.loaded = true;
+        state.loading = false;
+        this.scheduleRerender();
+      });
+  },
+
+  html(attrs, state) {
+    if (!state.loaded) {
+      this.refreshReviewableCount(state);
```

Comment:
Not sure I like calling this from inside the widget, but I don't see how else I can refresh the counts whenever the hamburger menu is open. Plus, we use a similar "workflow" for notifications.

Reply:
It is not amazing but probably OK based on our current structure and we can live with it.